### PR TITLE
Fixes #36560 - Persist pxe_loader on os_select

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

File: webpack/assets/javascripts/foreman_hosts.js [hunk 1]
```diff
@@ -35,6 +35,7 @@ export function getAttributesToPost(componentType) {
       'location_id',
       'ptable_id',
       'medium_id',
+      'pxe_loader',
```
Comment (MariaAga) [inline webpack/assets/javascripts/foreman_hosts.js:38]: Please add this word to `.eslintrc`

**Comment by pnovotny** [comment]
Verified the PR build foreman-3.8.0-0.20230630115616879905.pr9767.8116.gab339257f.el8.noarch on Sat. stream.

When changing the Lifecycle Environment on Host Group tab, the "PXE loader" value on Operating System tab does not reset.

**Comment by ofedoren** [comment]
Just a question related to tracking system, can we use `Refs` on tickets which get into branched/released version? It would seem that this is landed into 3.7.0, but apparently it will land into 3.7.1. Unless I missed dates for GA...

**Comment by adamruzicka** [comment]
Filed a separate issue, amended the commit message, force pushed, but github seems to be confused

**Review by ofedoren — APPROVED** [review]
Thanks, @adamruzicka and @MariaAga, LGTM and fixes the issue. The actual commit message corresponds with the new ticket, all good now.

**Pull request merged**